source/bundling-a-java-app.html.haml.markdown

```diff
@@ -16,32 +16,32 @@
   <span class="unselectable">$ </span>tar xvf SweetHome3D-5.2-linux-x64.tgz
   </pre>
 
-  ### Install the Flatpak Freedesktop SDK and runtime
+  ## Install the Flatpak Freedesktop SDK and runtime
   Every Flatpak application is required to specify a runtime that provides the core dependencies when the application is run. A Flatpak SDK is used just by you, the developer when building and bundling the application. See the [Developer page](developer.html) for further details on how the runtimes and SDKs work.
 
   For this example of bundling up Sweethome3D, we are going to use the Freedesktop SDK and runtime. The Freedesktop SDK and runtime are available in from the [GNOME SDK repository](https://sdk.gnome.org/). If you haven't previously set up this repository, you need to download the GPG key, then enable the repository:
 
   <pre>
   <span class="unselectable">$ </span>wget https://sdk.gnome.org/keys/gnome-sdk.gpg
-  <span class="unselectable">$ </span>flatpak remote-add --gpg-import=gnome-sdk.gpg sdk.gnome.org https://sdk.gnome.org/repo/
+  <span class="unselectable">$ </span>flatpak --user remote-add --gpg-import=gnome-sdk.gpg gnome https://sdk.gnome.org/repo/
```

Comment:
Oh, you don't need --user here anymore. These days we have policykit authenticated system-wide remotes. 
Just use --user for the things you locally build (like your app).